podcasts/FeatureFlag.swift

```diff
@@ -46,42 +46,51 @@ enum FeatureFlag: String, CaseIterable {
     /// Enable retrieving episode artwork from the RSS feed
     case episodeFeedArtwork
 
+    /// Enable the ability to rate podcasts
+    case giveRatings
+
     var enabled: Bool {
         if let overriddenValue = FeatureFlagOverrideStore().overriddenValue(for: self) {
             return overriddenValue
         }
 
+        return `default`
+    }
+
+    var `default`: Bool {
         switch self {
         case .freeTrialsEnabled:
-            return true
+            true
         case .tracks:
-            return true
+            true
         case .tracksLogging:
-            return false
+            false
         case .firebaseLogging:
-            return false
+            false
         case .endOfYear:
-            return false
+            false
         case .signInWithApple:
-            return true
+            true
         case .onboardingUpdates:
-            return true
+            true
         case .newSearch:
-            return true
+            true
         case .bookmarks:
-            return false
+            false
         case .discoverFeaturedAutoScroll:
-            return true
+            true
         case .patron:
-            return false
+            false
         case .showRatings:
-            return true
+            true
         case .autoplay:
-            return true
+            true
         case .newShowNotesEndpoint:
-            return true
+            true
         case .episodeFeedArtwork:
-            return Self.isTestFlight ? true : false
+            Self.isTestFlight ? true : false
+        case .giveRatings:
+            false
```

Comment:
Since we're touching this, what about adding a default case for all the true items, so we can reduce the list a bit?

Reply:
I wonder if this list should be there (and big) to bother us. There are a lot of flags there that need to be deleted.

But I don't have strong feelings against doing that — I have a small concern we might mess up and enable something by mistake but that shouldn't really happen.

Reply:
Makes sense. Lets leave it for now.